src/rdkafka.h

```diff
@@ -1385,6 +1496,14 @@ void rd_kafka_conf_set_dr_cb(rd_kafka_conf_t *conf,
  *
  * An application must call rd_kafka_poll() at regular intervals to
  * serve queued delivery report callbacks.
+ *
+ * The broker-assigned offset can be retrieved with \c rkmessage->offset
+ * ((if `producer.offset.report` is true), and the timestamp can be
```

Comment:
the property was `produce.offset.report` and it no longer exists.

Reply:
`produce.offset.report` is also referred to in a comment in `rd_kafka_message_s`